.travis.yml

```diff
@@ -175,16 +175,24 @@ after_success:
   fi;
 
 # This creates and uploads the gatk zip file to the nightly build bucket, only keeping the 10 newest entries
-# This also pokes the Dockerhub web api which is securely stored in the travis API with the variable "DOCKERHUB_URL"
+# This also constructs the Docker image and uploads it to https://cloud.docker.com/u/broadinstitute/repository/docker/broadinstitute/gatk-nightly/
 - if [[ $TRAVIS_BRANCH == master && $TRAVIS_EVENT_TYPE == cron && $UPLOAD == true ]]; then
+    docker login --username=$DOCKER_SERVICE_LOGIN --password=$DOCKER_SERVICE_PASS;
     $GCLOUD_HOME/gcloud components -q update gsutil;
     gsutil ls -l gs://gatk-nightly-builds | grep gatk | sort -r -k 2 | grep -o '\S\+$' | tail -n +11 | xargs -I {} gsutil rm {};
     ./gradlew bundle;
-    ZIP_FILE="$(ls build/ | grep .zip)";
+    ZIP_FILE="$(ls build/ | grep '^gatk.*.zip$' | grep -iv python)";
```

Comment:
One other option... we could make building /pushing the docker a gradle command.  Probably best saved for another time though.

Reply:
Thats not a bad idea. There are some nuances that wold be a bit of a pain to resolve